apps/server/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
missing repo-graph dependency, but graph is imported in `src/controller/Graph.ts`